Apps/Apps.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Not sure what the protocol here is, but does the `Package.resolved` need to be committed? If not, we should remove this file and `Package.resolved` to `.gitignore`

Reply:
It's okay for it to be committed. It makes it so that we can control *when* to update dependencies once their version requirements are specified more flexibly.